### ISSUE TITLE
Milling not just the outline but the whole surface

### ISSUE THREAD

**Issue by hudi84** [cammill/cammill#66]
Hi all!

I am new to milling and cammill too. So maybe this is a totally stupid question...

If I import a DXF either created by Inkscape or by OpenSCAD cammill treats the objects always as outline only. Is there a way to tell cammill that I want to mill the whole surface inside the outline of the object?

Any help is appreciated!

Thanks
hudi

**Comment by thatch** [comment]
Yes, at a minimum you need to configure `Milling -> Pocket = On` for the object, and if it thinks that object is an Outside you probably want to set `Milling -> Offset = Inside`.  You can then set `Milling -> Depth = 0` if you want to do a facing operation referenced to wherever you touch off.

I generally include another object that encompasses my entire drawing, and set that object to not use (I think unchecking it in the GUI does the same thing).  I tried following the GUI steps just now and it did not enable pocket mode, which is probably a bug; these are the lines that enable it in config (except for the include, that's a kind of preprocessor): https://github.com/statewave/cnc-homeshop/blob/master/_lib/cam/pocket.config
